test/administrator_test.rb

```diff
@@ -8,7 +8,7 @@ class AdministratorTest < Test::Unit::TestCase
 
     should "get a administrator by email address" do
       email = "admin@example.com"
-      stub_get(@auth, "admins.json?email=#{CGI.escape(email)}", "admin_details.json")
```

Comment:
I replaced `CGI.escape` with `ERB::Util.url_encode` because the former encodes a space as `+` while the latter encodes it as `%20` and `HTTParty` internally uses the latter internally. `CGI.escape` was failing some new tests.